src/components/pages/Map/_AnalyseLocations.jsx

```diff
@@ -20,16 +20,34 @@ export default class AnalyseLocations extends React.Component {
           columns={this.props.columns}
           data={this.props.data}
           pageSize={20}
-          filters
-          onSelectedRows={rows => console.info(rows)}
+          actions={{
+            showable: false,
+            editable: false,
+            removable: true
+          }}
+          pagination={{
+            enabled: true,
+            pageSize: 20,
+            page: 0
+          }}
+          onToggleSelectedRow={(ids) => {
```

Comment:
I think this name is confusing and not quite descriptive; "onRowSelect" or something similar would be more suitable.